app/api/resources/user.py

```diff
@@ -129,7 +129,14 @@ def get(cls):
     @users_ns.doc("update_user_profile")
     @users_ns.expect(auth_header_parser, update_user_request_body_model)
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.NAME_INPUT_BY_USER_IS_INVALID)
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        "%s\n%s"
+        % (
+            messages.NAME_INPUT_BY_USER_IS_INVALID,
```

Comment:
I think `NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT` message is enough there is no need of `NAME_INPUT_BY_USER_IS_INVALID`. Am I missing something else @devkapilbansal ?